src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/MainAccordion.tsx

```diff
@@ -42,16 +43,28 @@ type Props = {
   updateEntityUI: (uIstate: { [key: string]: JSONValue }) => void;
```

Comment:
can we have stricter types for `sidebarUIState` by the way? There's no way at the moment to know what we are expecting to be there.

Reply:
That's done